package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "metrics-automation-reporter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

We can include audit and lint checks to include on the pre-commit:
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
    "audit": "npm audit",
    "audit:fix": "npm audit fix",
    "lint:fix": "eslint . --fix",
    "lint": "eslint ."
  },
```

---

added